### ISSUE TITLE
Cannot build with Java 8, despite the documentation recommending it

### ISSUE THREAD

**Issue by mhmdanas** [00-Evan/shattered-pixel-dungeon#1094]
https://github.com/00-Evan/shattered-pixel-dungeon/blob/master/docs/getting-started-desktop.md#quick-setup states (emphasis mine):

> Ensure a [JDK (Java Development Kit)](https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html) is installed on your computer. (<b>Java 8 is recommended</b>)

However, running `./gradlew desktop:release` on the 1.3.2 source code using Java 8 spews out this error:

```
A problem occurred evaluating project ':android'.
> Failed to apply plugin 'com.android.internal.application'.
   > Android Gradle plugin requires Java 11 to run. You are currently using Java 1.8.
     Your current JDK is located in  /usr/local/Cellar/openjdk@8/1.8.0+345/libexec/openjdk.jdk/Contents/Home/jre
     You can try some of the following options:
       - changing the IDE settings.
       - changing the JAVA_HOME environment variable.
       - changing `org.gradle.java.home` in `gradle.properties`.
 ```

Running the same command with Java 11 works just fine.

**Comment by 00-Evan** [comment]
The documentation is out of date, as you've noticed Java 11 is now required. Will fix this for v1.4.0's release

**Comment by 00-Evan** [comment]
Fixed in commit 3eaa0e314419f26c5ea216718806013733e47d8e
